.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+language: node_js
+node_js:
+  - "0.10"
+cache:
+directories:
+  - node_modules
+install:
+  - "travis_retry npm install grunt-cli"
```

Comment:
I don't think this is needed. From [the Travis docs](http://docs.travis-ci.com/user/build-environment-updates/2015-02-03/#Node.js-VM):

> grunt-cli is provided for pre-installed Node.js 0.8 and up (including io.js 1.1.0)